kubernetes/filters.go

```diff
@@ -184,3 +184,13 @@ func FilterByRoute(spec map[string]interface{}, protocols []string, service stri
 	}
 	return false
 }
+
+func FilterIstioObjectsForSelector(selector labels.Selector, allObjects []IstioObject) []IstioObject {
+	var istioObjects []IstioObject
```

Comment:
If the resulting array is to be serialized as json, it's probably safer to initialize this array `istioObjects := []IstioObject{}` ?

Reply:
It's not exposed but the initialization makes sense